src/rules/incognia-login.js

```diff
@@ -0,0 +1,67 @@
+/**
+ * @title Incognia Login Rule
```

Comment:
I believe this is for the Incognia Authentication integration, correct? If so, can we use the same name here and slug version for the file, like `incognia-authentication`?